app/ts/components/TransferPage/index.tsx

```diff
@@ -33,7 +33,7 @@ const MainPanel = () => {
 	const { transaction, data, send, clearData } = useTransfer()
 
 	const handleTokenSelect = (token?: TokenMeta) => {
```

Comment:
maybe better function name than `handle` This could be `updateTokenSelect` instead?

Reply:
changed to `setUserSelectedToken` to be more descriptive in commit [e185417](https://github.com/DarkFlorist/lunaria/pull/133/commits/e18541702e4d255983547a280bb996cf4762e738)